### PR TITLE
Favicon2

### PR DESCRIPTION
**Pull request by JoshMacSween**

### CODE REVIEW

**Comment by JoshMacSween** [comment]
closes issue #23

**Comment by alanvardy** [comment]
Build isn't passing. Check dem tests!

File: config/routes.rb [hunk 1]
```diff
@@ -4,8 +4,7 @@
 #                               invites GET    /invites(.:format)                                                                       invites#index
 #                                       POST   /invites(.:format)                                                                       invites#create
 #                            new_invite GET    /invites/new(.:format)                                                                   invites#new
-#                                invite GET    /invites/:id(.:format)                                                                   invites#show
```
Comment (alanvardy) [inline config/routes.rb]: Is this path no longer needed?

**Comment by alanvardy** [comment]
Can you restore the .env.example? You have deleted it in this commit.

**Comment by alanvardy** [comment]
I don't think that you have added the ruby image file.

File: config/routes.rb (hunk 1 above)
Comment (JoshMacSween) [reply config/routes.rb]: Odd.. I didn't touch it.

Comment (alanvardy) [reply config/routes.rb]: Haha, never mind.

**Comment by alanvardy** [comment]
Thanks Josh!

**Pull request merged**